src/amo/components/SearchForm.js

```diff
@@ -61,8 +77,12 @@ export class SearchFormBase extends React.Component {
   }
 }
 
-export function mapStateToProps({ api }) {
-  return { api };
+export function mapStateToProps(state) {
+  return {
+    addonType: validAddonTypes.includes(state.viewContext.context) ?
```

Comment:
We often want to know if an addonType is passed to the component, and the keys passed around are import to methods that make up our query strings/API requests, etc.

We often need to know not only what the addonType is but also if there *is* an addonType.